master_check.do

```diff
@@ -14,14 +14,14 @@ ipadoheader, version(15.0)
    ================== Import globals from Excel  ================= 
    =============================================================== */
 
-ipacheckimport using "04_checks/01_inputs/hfc_inputs.xlsm"
+ipacheckimport using "../04_checks/01_inputs/hfc_inputs.xlsm"
 
 
 /* =============================================================== 
    ==================== Replace existing files  ================== 
    =============================================================== */
 
-foreach file in "${outfile}" "${enumdb}" "${textaudit}" "${researchdb}" "${bcfile}"{
+foreach file in "${outfile}" "${enumdb}" "${researchdb}" "${bcfile}" "${progreport}" "${dupfile}"{

```

Comment:
I think these are old too @rsandino

Reply:
progreport and dupfile do need to be in there so they don't overwrite but the removal of textaudit was unintentional. I'll fix that now